backend/common/models.py

```diff
@@ -99,13 +100,16 @@ class RequestType(Enum):
 
 
 class Request(BaseModel):
-    id: str
-    group_id: str
-    sender_id: str
-    recipient_id: str
-    date: datetime
-    type: RequestType
-    status: RequestStatus = RequestStatus.PENDING
+    id: Optional[PydanticObjectId] = Field(alias="_id", default=None)
+    group_id: PydanticObjectId
+    sender_id: PydanticObjectId
+    recipient_id: PydanticObjectId
+    date: datetime = Field(default_factory=datetime.now)
```

Comment:
The date should not be a field. This attribute will be populated at object initialisation